frontend/containers/share-icons/component.tsx

```diff
@@ -0,0 +1,118 @@
+import { FC, useState } from 'react';
+
+import { Facebook, Link, Mail, Twitter } from 'react-feather';
+import { FormattedMessage, useIntl } from 'react-intl';
+
+import classNames from 'classnames';
+
+import { useRouter } from 'next/router';
+
+import Button from 'components/button';
+import Icon from 'components/icon';
+import Toast from 'components/toast';
+
+import { ShareIconsProps } from '.';
+
+export const ShareIcons: FC<ShareIconsProps> = ({ title }) => {
+  const [copied, setCopied] = useState(false);
+  const { formatMessage } = useIntl();
+
+  const { asPath } = useRouter();
+  // Facebook don't support locales, so we use the staging url when in local
+  const shareLinkUrl = process.env.NEXT_PUBLIC_PROXY_BACKEND
+    ? `https://staging.hecoinvest.org${asPath}`
+    : `${process.env.NEXT_PUBLIC_FRONTEND_URL}${asPath}`;
+
+  const isProjectPage = asPath.includes('/project/');
+
+  const shareLinks = {
+    facebook: `https://www.facebook.com/sharer/sharer.php?u=${shareLinkUrl}`,
+    twitter: `https://twitter.com/intent/tweet?url=${shareLinkUrl}`,
+    email: `mailto:?subject=${formatMessage(
+      {
+        defaultMessage: 'Someone has shared a HeCo invest {shared} with you',
+        id: 'wD2sM8',
+      },
+      { shared: isProjectPage ? 'project' : 'contact' }
```

Comment:
“project” and “contact” need to be translated. I think it would be more robust (for the translations) if we'd duplicate the entire strings:
- “Someone has shared a HeCo invest project with you”
- “Someone has shared a HeCo invest contact with you”